### DISCUSSION TITLE
What does this do

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#200] [General]
What is this?
It's use?

**Comment by xItzDiamondx** [comment]
Technically just has little lines of code that you bookmark. Then when you need it, you click on it while on Scratch. Email me or Shluffy for any further questions.
shluffymonster@outlook.com
xitzdiamondx@protonmail.ch

**Reply by Quantum-Codes** [reply]
What lines? Uhm.. I can't email tho

**Reply by Shluffy** [reply]
Let me try to fix a bug then I will explain it to you

**Reply by Quantum-Codes** [reply]
Sure

**Comment by Shluffy** [comment]
@Quantum-Codes this contains bookmarklets that you can copy, and click on on the scratch site to do certain things. Go to https://scratch-bookmarklets.github.io/bookmarklets.html to find them

**Reply by Quantum-Codes** [reply]
Ohhh! How to use them?? Why they keep repeating in list?

**Reply by imlens** [reply]
@Quantum-Codes That's just a little bug.

**Reply by Quantum-Codes** [reply]
Oh ok
